test/ConvertersTests.ts

```diff
@@ -82,4 +83,48 @@ describe('Converters', () => {
     expect(bindingData.Sys).to.be.undefined;
     expect(bindingData.sys.UtcNow).to.be.undefined;
   });
+
+  it('catologues binding definitions', () => {
+    let functionMetaData: rpc.IRpcFunctionMetadata = <rpc.IRpcFunctionMetadata> {
+        name: "MyFunction",
+        directory: ".",
+        scriptFile: "index.js",
+        bindings: {
+            req: {
+                type: "httpTrigger",
+                direction: rpc.BindingInfo.Direction.in
+            },
+            res: {
+                type: "http",
+                direction: rpc.BindingInfo.Direction.out
+            },
+            firstQueueOutput: {
+                type: "queue",
+                direction: rpc.BindingInfo.Direction.out
+            },
+            secondQueueOutput: {
+                type: "queue",
```

Comment:
Add
- binding with empty direction
 - binding without direction

Reply:
@pragnagopa - is that a possible case? I thought that there would always be a binding defined ([docs seem to say so](https://docs.microsoft.com/en-us/azure/azure-functions/functions-triggers-bindings#binding-direction), although i'm sure that could change!)

Reply:
For unit testing we need test all possibilities 😊

Reply:
true true :) will add!